### PR TITLE
Removed Patient type from Metadata classes.

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

File: smart-cards/src/main/java/gov/va/api/health/smartcards/MetadataController.java [hunk 1]
```diff
@@ -44,7 +44,6 @@
   private static List<CapabilityStatement.CapabilityResource> resources() {
     return Stream.of(
             SupportedResource.builder()
```
Comment (jblefkove-va) [inline smart-cards/src/main/java/gov/va/api/health/smartcards/MetadataController.java:46]: the whole resource needs to be removed

Comment (apeirats-va) [reply smart-cards/src/main/java/gov/va/api/health/smartcards/MetadataController.java:46]: Should this return Parameters instead?

Comment (jblefkove-va) [reply smart-cards/src/main/java/gov/va/api/health/smartcards/MetadataController.java:46]: yeah that makes more sense

Comment (that-guy-is-it) [reply smart-cards/src/main/java/gov/va/api/health/smartcards/MetadataController.java:46]: 🤦 On it

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**